docs/submission/submitting-molecular-data.md

```diff
@@ -94,7 +155,7 @@ If the data for a sample is different than what has been registered, metadata va
 - `file_r2`: Name of the sequencing file containing reads from the second end of a paired end sequencing run. Required if and only if paired end sequencing was done.
 - `read_length_r1`: Length of sequencing reads in `file_r1`; this corresponds to the number of sequencing cycles of the first end.
 - `read_length_r2`: Length of sequencing reads in `file_r2`; this corresponds to the number of sequencing cycles of the second end.
-- `insert_size`: For paired end sequencing the average size of sequences between two sequencing ends. Required for paired end sequencing.
+- `insert_size`: For paired end sequencing, the average size of sequences between two sequencing ends. Required for paired end sequencing.
 - `sample_barcode`: According to the SAM specification, this is the expected barcode bases as read by the sequencing machine in the absence of errors.
 - `**library_name`: Name of a sequencing library made from a molecular sample or a sample pool (multiplex sequencing).
 

```

Comment:
To make it more accurate, according to schema definition (https://github.com/icgc-argo/argo-metadata-schemas/blob/0.3.0/schemas/sequencing_experiment.json#L116-L126):
            "library_name",
            "platform_unit",
            "is_paired_end",
            "read_length_r1",
            "file_r1",
            "insert_size",
            "read_length_r2",
            "file_r2",
            "sample_barcode",
            "submitter_read_group_id"
are all required fields. 
That means the fields have to be included into the payload. 

However fields:  `insert_size`, `read_length_r1`, `read_length_r2`, `file_r2`, `sample_barcode` could have value of `null`.